src/Exception/ExceptionInterface.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+declare(strict_types=1);
+
 namespace Laminas\DeveloperTools\Exception;
 
 interface ExceptionInterface
```

Comment:
Doesn't extend \Throwable

Reply:
Would that be a BC break? If users are using this interface and extend throwable themselves?

Reply:
Yes, a technical BC break but @Ocramius always lets me get away with it 😂

Reply:
ah ok, give me a mo I will add